app/controllers/blocked/users_controller.rb

```diff
@@ -12,7 +12,8 @@ class UsersController < ApplicationController
     # GET /users/:user_id/blocked/users
     def index
       @blocks = @user.blocks_as_blocker
-        .joins(:blocked).includes(blocked: :default_pseud)
+        .joins(:blocked)
+        .includes(blocked: { default_pseud: { icon_attachment: :blob }, pseuds: {} })
```

Comment:
Slightly prettier syntax:
```suggestion
        .includes(blocked: [:pseuds, default_pseud: { icon_attachment: :blob }])
```

Reply:
Ooh thanks, I was wondering if there was a good option for the mixed preloading